cashews/utils/bitarray.py

```diff
@@ -0,0 +1,29 @@
+class Bitarray:
+    __slots__ = ("_value",)
+
+    def __init__(self, value, base=10):
+        self._value = int(value, base)
+
+    def get(self, index: int, size: int = 1) -> int:
```

Comment:
It seems like the whole method can be done using bitwise operations. It should be faster and easier to understand

```python
def get(value, idx, size):
    res = 0
    for i in range(idx, idx + size):
        res <<= 1
        if value & (2 ** i):
            res |= 1
    return res
```